### ISSUE TITLE
Which version of Font Awesome is used?

### ISSUE THREAD

**Issue by DanAtShenTech** [alexgleason/wagtailfontawesome#28]
The link on your main github page is to the main FontAwesome page, which at this time is v. 5.  I needed to use the straight "H" icon ('heading' in FA v. 5).  On [this page](https://fontawesome.com/icons/heading?style=solid) it says:
> This icon replaces Font Awesome 4's fa-header.
We created a shim to help folks upgrade easily their existing projects from Font Awesome 4 to 5....

However, when I use `icon="fa-heading"`, no icon shows up; but when I use `icon="fa-header"`, then the "H" icon appears.  Something seems to be amiss here.

**Comment by FlipperPA** [comment]
This package still uses version 4.7. I started to look at the upgrade to 5, and it wasn't as trivial as I had hoped. I haven't had the cycles to dig deeper, and I don't think Alex has either. It's on the big list of things to do, however! If you want to take a stab at it, feel free. Thanks!

**Comment by alexgleason** [comment]
Yeah, I was pretty surprised by the release of Font Awesome 5. Font Awesome used to be a humble open source iconset, but they've been slowly commercializing it and bloating it with unnecessary icons. A lot of people wouldn't use FA4 because of its bloat. Personally, I felt it was a reasonable compromise as a general purpose tool. But in FA5 they seem to have really doubled down on it. I'm not sure how I want to handle it.

I like how Semantic UI [basically forked Font Awesome 4](https://semantic-ui.com/elements/icon.html) and maintained their own list. I might want to do something like that. Alternatively, we could release a new version of wagtailfontawesome that's based on FA5.

In the meantime, I think we should at least update the readme to point to the correct list of icons. Anyone know where to find that for FA4?

**Comment by FlipperPA** [comment]
One think I've done with [WagtailCodeBlock](https://github.com/FlipperPA/wagtailcodeblock) is switch my versioning to reflect the underlying library, PrismJS. For example, the latest release is version `1.11.0.0`,  which includes PrismJS version `1.11.0`. We could switch the current release to be version `4.7.x.0` and then create a new version, `5.0.x.0`, so people could easily pin to the version of FontAwesome they want to use. What do you think?

**Comment by DanAtShenTech** [comment]
@FlipperPA I like the idea of pinning wagtailfontawesome to a particular FA release (I've submitted a pull request).  Interesting that @alexgleason mentioned Semantic UI.  We use SUI here too.  It looks like SUI is now at FA 5 -- from [their site](https://semantic-ui.com/elements/icon.html):  Semantic includes a complete port of Font Awesome 5.0.6 designed by the FontAwesome team for its standard icon set.

**Comment by alexgleason** [comment]
I just found this great fork of Font Awesome 4.7 that might be worth considering instead of moving to 5.0

Fork Awesome: https://forkawesome.github.io/Fork-Awesome/

**Comment by SalahAdDin** [comment]
@alexgleason why?

**Comment by alexgleason** [comment]
I don't like the direction Font Awesome 5 is going at all and don't want to try implementing it. Fork Awesome is a community fork of FA4.7 meaning we'll be kept updated by a project maintainer while not having to use FA5
